src/core/traits/scalar.rs

```diff
@@ -278,8 +278,7 @@ impl FloatEx for f64 {
     }
     #[inline(always)]
     fn acos_approx(self) -> Self {
-        // TODO: clamp range
```

Comment:
I should look through my TODOs...

Reply:
If you are using VS Code you might wanna try https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree :)